lib/puppet/functions/default_content.rb

```diff
@@ -1,10 +1,16 @@
 Puppet::Functions.create_function(:default_content) do
-  # @summary Takes an optional content and an optional template name and
-  # returns the contents of a file.
-  # @param :content
-  # @param :template_name
+  # @summary Takes an optional content and an optional template name and returns the contents of a file.
```

Comment:
According to the style guide, you're not supposed to use the `@summary` tag.  It seems to work better than anything else I've tried so far though.